zkvm/src/vm.rs

```diff
@@ -287,6 +287,31 @@ where
         Ok(())
     }
 
+    fn const_instr(&mut self) -> Result<(), VMError> {
```

Comment:
would it be better or awful to use raw identifier syntax?

```suggestion
    fn r#const(&mut self) -> Result<(), VMError> {
```

https://rust-lang-nursery.github.io/edition-guide/rust-2018/module-system/raw-identifiers.html#raw-identifiers